duckdb_engine/__init__.py

```diff
@@ -17,14 +17,15 @@
 
 import duckdb
 import sqlalchemy
-from sqlalchemy import pool, text, util
+from sqlalchemy import pool, select, sql, text, util
 from sqlalchemy import types as sqltypes
 from sqlalchemy.dialects.postgresql import UUID
 from sqlalchemy.dialects.postgresql.base import (
     PGDialect,
     PGIdentifierPreparer,
     PGInspector,
     PGTypeCompiler,
+    pg_catalog,
```

Comment:
tests make it look like you are importing from the wrong place? perhaps it should be

`from sqlalchemy.dialects.postgresql import pg_catalog`?

See https://github.com/sqlalchemy/sqlalchemy/blob/7b12826a02fa1f83acd64bd531c9e47832d22cb0/lib/sqlalchemy/dialects/postgresql/pg_catalog.py

Reply:
Done in the latest commit!